### PR TITLE
Working group docs

### PR DESCRIPTION
**Pull request by rowanmanning**
Make a few changes to the Working Group documentation to better describe what we do. This also adds a link to the home page so that this documentation is more discoverable.

### CODE REVIEW

File: site/pages/working-group.md [hunk 1]
```diff
@@ -12,13 +12,22 @@ The engineering competencies have been planned, written and rolled out by a work
...
 
-Now that the competencies have stabilised, the Working Group meets once a week. We discuss any issues that have been opened since the last meeting, as well as planning future work. Tasks are assigned to individuals on a voluntary basis.
 
-[The agenda for this meeting is here](https://docs.google.com/document/d/1o2MfeMZjovFva-0BwSbm2UN_06HxxbhWfKPugmkNO1Q/edit) (only accessible to Working Group members).
+## Process
+
+The Working Group meets once every two weeks, and we work to this agenda. If tasks are assigned to individuals then it's on a voluntary basis, and non-critical tasks don't have deadlines. This allows Working Group members to pick things up when it can fit around their regular work.
 
-Outside of this weekly meeting, members do any work they can fit around their normal schedule.
+We follow an agenda for these meetings, make notes, and assign actions in [this document](TODO-agenda-link).
```
Comment (rowanmanning) [inline site/pages/working-group.md:30]: @apaleslimghost let me know when we have a new/public agenda document and I'll link it here.

File: site/pages/home.html [hunk 2]
```diff
@@ -52,6 +52,7 @@ <h2>Contributing</h2>
 			<ul>
 				<li><a href="{{site.github.repository_url}}/blob/master/CONTRIBUTING.md#contributing" class="o-typography-link--external">Read the contributing guide</a></li>
 				<li><a href="{{site.github.repository_url}}/issues/new" class="o-typography-link--external">Open an issue on GitHub</a></li>
+				<li><a href="{{site.github.repository_url}}/issues/new" class="o-typography-link--external">Read about the Engineering Progression Working Group</a></li>
```
Comment (apaleslimghost) [inline site/pages/home.html:55]: this is the wrong URL :) and it probably shouldn't have `o-typography-link--external`

File: site/pages/working-group.md (hunk 1 above)
Comment (apaleslimghost) [reply site/pages/working-group.md:30]: i should have time to put that together tomorrow

File: site/pages/home.html (hunk 2 above)
Comment (rowanmanning) [reply site/pages/home.html:55]: derpaderp

File: site/pages/working-group.md (hunk 1 above)
Comment (rowanmanning) [reply site/pages/working-group.md:30]: No rush 🙂

Comment (rowanmanning) [reply site/pages/working-group.md:30]:
```suggestion
We follow an agenda for these meetings, make notes, and assign actions in [this document](https://docs.google.com/document/d/1GHtXdANM6Yq6Xwh_6lHi3mTY-BtRF-0iPcTWFhfAnL4/edit).
```

**Review by apaleslimghost — APPROVED** [review]

**Pull request merged**